backend/infrahub/graphql/queries/account.py

```diff
@@ -97,12 +98,16 @@ async def resolve_account_permissions(
         raise ValueError("An account_session is mandatory to execute this query")
 
     fields = await extract_fields_first_node(info)
-    permissions = await fetch_permissions(
-        db=context.db, branch=context.branch, account_id=context.account_session.account_id
-    )
+    permissions: dict[str, list[GlobalPermission]] = {}
+    if registry.permission_backends:
```

Comment:
As permissions_backends is a list I think we can skip the if-statement and just go with the for loop.